source/_integrations/template.markdown

```diff
@@ -592,6 +592,25 @@ template:
 
 {% endraw %}
 
+### State based select - Control Day/Night mode of a camera
+
+This show how a state based template select can be used to call a service.
+
+```yaml
+template:
+  select:
+    - name: "Porch Camera Day-Night Mode"
+      unique_id: porch_camera_day_night_mode
+      state: "{{ state_attr('camera.porch_camera_sd', 'day_night_mode') }}"
+      options: "{{ ['off', 'on', 'auto'] }}"
```

Comment:
Do we really require a template here? Like this doesn't work? 🤔 

```suggestion
      options: 
        - "off"
        - "on"
        - "auto"
```

Reply:
Right, that doesn't work, hence the example.